src/utils/espresso/sum_check/verifier.rs

```diff
@@ -9,24 +9,27 @@
 
 //! Verifier subroutines for a SumCheck protocol.
 
-use super::{SumCheckSubClaim, SumCheckVerifier};
-use crate::utils::virtual_polynomial::VPAuxInfo;
+use super::{
+    structs::{IOPProverMessage, IOPVerifierState},
+    SumCheckSubClaim, SumCheckVerifier,
+};
+use crate::{transcript::Transcript, utils::virtual_polynomial::VPAuxInfo};
+use ark_ec::CurveGroup;
 use ark_ff::PrimeField;
 use ark_std::{end_timer, start_timer};
 
-use super::structs::{IOPProverMessage, IOPVerifierState};
 use espresso_subroutines::poly_iop::prelude::PolyIOPErrors;
 use espresso_transcript::IOPTranscript;
 
 #[cfg(feature = "parallel")]
 use rayon::iter::{IndexedParallelIterator, IntoParallelIterator, ParallelIterator};
 
-impl<F: PrimeField> SumCheckVerifier<F> for IOPVerifierState<F> {
-    type VPAuxInfo = VPAuxInfo<F>;
-    type ProverMessage = IOPProverMessage<F>;
-    type Challenge = F;
-    type Transcript = IOPTranscript<F>;
-    type SumCheckSubClaim = SumCheckSubClaim<F>;
+impl<C: CurveGroup> SumCheckVerifier<C> for IOPVerifierState<C> {
+    type VPAuxInfo = VPAuxInfo<C::ScalarField>;
+    type ProverMessage = IOPProverMessage<C::ScalarField>;
+    type Challenge = C::ScalarField;
+    type Transcript = IOPTranscript<C::ScalarField>;
```

Comment:
I think that we can now remove this `IOPTranscript` since is not used. This includes also removing the `espresso_transcript` dependency from the repo (https://github.com/privacy-scaling-explorations/folding-schemes/blob/feature/sumcheck/Cargo.toml#L23).

Reply:
Ah yes, good catch! Removed it [here]([812d710](https://github.com/privacy-scaling-explorations/folding-schemes/pull/40/commits/812d710cc2cf54b761845cb95c266c6d418659de))